tests/unit/amo/components/TestCollection.js

```diff
@@ -572,6 +594,58 @@ describe(__filename, () => {
     expect(wrapper.find(AddonsCard)).toHaveProp('editing', false);
   });
 
+  it('sets a default placeholder count', () => {
+    const wrapper = renderComponent();
+    expect(wrapper.find(AddonsCard)).toHaveProp('placeholderCount', 5);
```

Comment:
If we made this a `const` and exported it then we could use the value from the component here, rather than hardcoding `5`.